source/synthDrivers/espeak.py

```diff
@@ -90,8 +99,69 @@ def _processText(self, text):
 			0x5B: u" [", # [: [[ indicates phonemes
 		})
 
-	def speak(self,speechSequence):
-		defaultLanguage=self._language
+	def _determineLangFromCommand(self, command: LangChangeCommand) -> Optional[str]:

```

Comment:
Perhaps rename to `normalizeLangChangeCommand`? Though then consider returning / modifying the `langChangeCommand` in place.

Reply:
This has been changed in 5c3388b